libs/features/mas/web-components/src/global.css.js

```diff
@@ -25,6 +24,8 @@ styles.innerHTML = `
     --consonant-merch-card-cta-font-size: 15px;
 
     /* headings */
+    --merch-card-heading-xxs-font-size: 16px;
```

Comment:
these changes that are supposed to be shipped as part of @Axelcureno's PRs, if shipped earlier, might cause regression.
I recommend to met his PRs first.

Reply:
the other way around i would say